test/unit/image/pixels.js

```diff
@@ -42,6 +42,39 @@ suite('pixels', function() {
     test('works with integers', function() {
       assert.deepEqual(img.get(25, 25), [255, 0, 0, 255]);
       assert.deepEqual(img.get(25, 26), [0, 0, 255, 255]);
+      assert.deepEqual(img.get(0, 0), [255, 0, 0, 255]);
+      assert.deepEqual(img.get(49, 49), [0, 0, 255, 255]);
+    });
+
+    test('works when out of bounds', function() {
+      assert.deepEqual(img.get(25, -1), [0, 0, 0, 0]);
+      assert.deepEqual(img.get(-1, 26), [0, 0, 0, 0]);
+      assert.deepEqual(img.get(25, 50), [0, 0, 0, 0]);
+      assert.deepEqual(img.get(50, 26), [0, 0, 0, 0]);
+    });
+
+    test('get() works', function() {
+      var copy = img.get();
+      assert.instanceOf(copy, p5.Image);
+      assert.equal(copy.width, img.width);
+      assert.equal(copy.height, img.height);
+
+      assert.deepEqual(copy.get(25, 25), [255, 0, 0, 255]);
+      assert.deepEqual(copy.get(25, 26), [0, 0, 255, 255]);
+      assert.deepEqual(copy.get(0, 0), [255, 0, 0, 255]);
+      assert.deepEqual(copy.get(49, 49), [0, 0, 255, 255]);
+    });
+
+    test('get(x,y,w,h) works', function() {
+      for (var w = 1; w < img.width + 5; w += 2) {
+        for (var x = -w * 2; x <= img.width + w * 2; x += 4) {
+          var copy = img.get(x, x, w, w);
+          assert.instanceOf(copy, p5.Image);
+          assert.equal(copy.width, w);
+          assert.equal(copy.height, w);
+          assert.deepEqual(copy.get(0, 0), img.get(x, x));
+        }
+      }
```

Comment:
Wouldn't it be better if there's a test for get(x,y) too?

Reply:
This is present in several places here unless I am mistaken.

Reply:
they are, but they are lumped in with another test case. it took me a while to find them. going request they be a separate `it` block

Reply:
Ok, these tests are literally littered with `get(x,y)` :sweat_smile: but wouldn't it be better if there was an explicit test for `get(x,y)`, something which @outofambit also suggests?

Reply:
i think the `works with integers` and `works when out of bounds` tests cover this.

Reply:
ah ok! cool. i'm going to suggest a couple edits to make that clearer. thanks!